qa/L0_cmdline_trace/test.sh

```diff
@@ -618,11 +618,140 @@ set -e
 kill $SERVER_PID
 wait $SERVER_PID
 
+set +e
```

Comment:
Random nit: but I also think it would help organization if `L0_cmdline_trace` was renamed to `L0_trace_cmdline` - if you feel like tweaking that at some point, doesn't have to be in this PR.